css-overflow-4/Overview.bs

```diff
@@ -707,12 +707,15 @@ Indicating Block-Axis Overflow: the 'block-ellipsis' property</h3>
 	</pre>
 
 	This property allows inserting content into the last line box
-	before a (forced <em>or</em> unforced) <a>region break</a>
-	to indicate the continuity of truncated/interrupted content.
+	before a (forced <em>or</em> unforced) <a>region break</a> or a [=clamp point=]
+	to indicate the continuity of truncated / interrupted content.
 	It only affects line boxes contained directly by the <a>block container</a> itself,
 	but as it inherits, will have an effect on descendants’ line boxes unless overridden.
-	If the box contains no line box immediately preceding a <a>region break</a>,
-	then this property has no effect.
+
+	This property only affects a line box if it immediately precedes
+	eiher a [=region break=] or a [=clamp point=] in the [=block formatting context=].
+	If the [=clamp point=] is placed at the end of the [=line-clamp container=], then the line will
+	not be affected.
```

Comment:
Editorial
```suggestion
	This property only affects a line box
	if it immediately precedes eiher a [=region break=]
	or a [=clamp point=] in the [=block formatting context=].
	If the [=clamp point=] is placed at the end of the [=line-clamp container=],
	then the line will not be affected.
```
Reason: this spec uses semantic line-breaks, and I'd prefer to stay consistent with that. See https://rhodesmill.org/brandon/2012/one-sentence-per-line/